coverage_config_x86_64.json

```diff
@@ -0,0 +1,5 @@
+{
+    "coverage_score": 33.3, 
```

Comment:
Why 33.3 if we have no tests? I also just ran `find . -type f -name "test_commit_format.py" | xargs pytest` in container and the reported value seems to be 0.

Reply:
Hmm, looks like it depends on the Rust version; from 1.49.0 the reported coverage is 33.3 and before -> 0. Anyway it's not related to the changes in this PR, so disregard my comment.